test/upgrade/prober/wathola/config/defaults.go

```diff
@@ -69,7 +72,7 @@ func defaultValues() *Config {
 		Sender: SenderConfig{
 			Address:  fmt.Sprintf("http://localhost:%v/", forwarderPort),
 			Interval: 10 * time.Millisecond,
-			Cooldown: time.Second,
+			Cooldown: 100 * time.Millisecond,
```

Comment:
I was thinking about this.

Let's remove the `Cooldown` option, at least for now. Eventing shouldn't require this circuit breaking to recover. So, it should be okay to continue sending events, while it's not responding. 

Also, I'm thinking that, if such circuit breaking may be required for ex.: when using a high amount of data this simple static value of cooldown will not be enough. Maybe then, we would be needing a better solution of exponential round up, that might be based on the interval parameter.

Before that some other things needs to be fixed first: concurrency for sender, [Coordinated Omission Problem](https://jaxenter.com/performance-problems-pre-production-tests-114128.html) fix.

So, I propose to remove `Cooldown` for now. It will simplify calculation of unavailability periods.

And then, maybe in future we will reintroduce a proper circuit breaking with proper calculation of unavailability periods.

Reply:
I think removing the `Cooldown` makes sense to me in this case as the sender is already quite slow anyway because of waiting for http response. 

But I don't think it simplifies the unavailability period calculation though. The only accurate way is to use a real timer so that when the send failed, we start the timer, and reset the timer when the event is sent. My experiment with gcp broker is that we don't see any unavailability at all, so the accuracy might not be very important in the sense that a good broker should try to achieve 0 unavailability during upgrade.

Reply:
I agree. We should have a timer, and reset it. Then we will report real time.

Reply:
OK, now we are using timer to get the real period, cooldown is removed.